### PR TITLE
[WIP] dotCMS/core#14513 Breadcurmb

### PR DESCRIPTION
**Pull request by fmontes**

### CODE REVIEW

File: src/app/api/services/dot-router/dot-router.service.ts [hunk 1]
```diff
@@ -82,6 +82,10 @@ export class DotRouterService {
         this.router.navigate(['/c/site-browser']);
     }
 
+    goToEditContentType(id: string) {
```
Comment (alfredo-dotcms) [inline src/app/api/services/dot-router/dot-router.service.ts]: add `void`

File: src/app/view/components/dot-crumbtrail/service/dot-crumbtrail.service.ts [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                    });
+                }
+
+                return crumbTrail;
+            })
+        );
+    }
+}
+
+export interface Crumb {
```
Comment (fmontes) [inline src/app/view/components/dot-crumbtrail/service/dot-crumbtrail.service.ts]: DotCrumb?

Comment (fmontes) [reply src/app/view/components/dot-crumbtrail/service/dot-crumbtrail.service.ts]: Export?

File: src/app/view/components/dot-navigation/services/dot-navigation.service.ts [hunk 3]
```diff
@@ -208,6 +208,10 @@ export class DotNavigationService {
         this.setMenu(updatedMenu);
     }
 
+    onNavigationEnd(): Observable <Event> {
```
Comment (fmontes) [inline src/app/view/components/dot-navigation/services/dot-navigation.service.ts]: Merge master, this already exist.

File: src/app/view/components/dot-crumbtrail/service/dot-crumbtrail.service.ts [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+        router: Router,
+        private activeRoute: ActivatedRoute
+    ) {
+        this.dotNavigationService.onNavigationEnd().pipe(
+            map((event: NavigationEnd) => event.url),
+            filter((url: string) => !this.URL_EXCLUDES.includes(url)),
+            switchMap(this.getCrumbtrail.bind(this))
+        ).subscribe((crumbTrail: Crumb[]) => this.crumbTrail.next(crumbTrail));
+
+        this.getCrumbtrail(router.url).subscribe(crumbTrail => this.crumbTrail.next(crumbTrail));
```
Comment (alfredo-dotcms) [inline src/app/view/components/dot-crumbtrail/service/dot-crumbtrail.service.ts]: add type to `crumbTrail`

File: src/app/view/components/dot-crumbtrail/dot-crumbtrail.component.spec.ts [hunk 5]
```diff
@@ -0,0 +1,73 @@
+/* tslint:disable:no-unused-variable */
```
Comment (alfredo-dotcms) [inline src/app/view/components/dot-crumbtrail/dot-crumbtrail.component.spec.ts]: remove

**Pull request merged**